packages/cli/src/core/plugin.js

```diff
@@ -6,6 +6,22 @@ const HttpMock = require("./http-mock");
 const Collection = require("./collection");
 const path = require("path");
 const Coverage = require("./coverage");
+const {getPackageJson} = require("../utils/fs");
```

Comment:
I would suggest that we put this logic in the config class: https://github.com/restqa/restqa/blob/master/packages/cli/src/config/index.js#L236

It will isolate the logic and it will be easier to test as well

Reply:
Yeah we also need to write tests, it's in the plan ✋🏼